internal/pkg/manifest/backend_svc.go

```diff
@@ -52,6 +53,9 @@ func NewBackendService(props BackendServiceProps) *BackendService {
 	svc.BackendServiceConfig.ImageConfig.Build.BuildArgs.Dockerfile = stringP(props.Dockerfile)
 	svc.BackendServiceConfig.ImageConfig.Port = uint16P(props.Port)
 	svc.BackendServiceConfig.ImageConfig.HealthCheck = props.HealthCheck
+	if props.Platform != nil {
+		svc.BackendServiceConfig.Platform = props.Platform
+	}
```

Comment:
Is it equivalent to
```suggestion
	svc.BackendServiceConfig.Platform = props.Platform
```
right now?

Reply:
Without the if-statement, the default behavior (when there isn't a platform redirection) would make the whole Platform field nil, not just the member fields. So if we want the field to always be set as default, then we should leave the if-statement in.

Reply:
but what's the difference between platform and [healthcheck](https://github.com/aws/copilot-cli/pull/2814/files#diff-2147c7bfea12eb3dff6a866c245ac1ed73e671d7c47bf06ff906e9dfc741d380R55)?